src/providers/MeetingProvider/MeetingManager.ts

```diff
@@ -109,8 +109,8 @@ export class MeetingManager implements AudioVideoObserver {
   private eventDidReceiveRef: EventObserver;
 
   private deviceLabels: DeviceLabels | DeviceLabelTrigger;
-  
-  public getDeviceLabels() : DeviceLabels | DeviceLabelTrigger {
```

Comment:
Removed accessibility modifier `public` because we have below eslint rule:
https://github.com/aws/amazon-chime-sdk-component-library-react/blob/b3d5389686419758cd2a327399f93098539690e0/.eslintrc.json#L62-L70